Magento_Contact/styles/modules/_button.scss

```diff
@@ -0,0 +1,3 @@
+.button {
+    margin: 5px 0;
+}
```

Comment:
you do not change exact component styles in theme repo b/c it will influence all buttons on the website. To delete.